styles/index.css

```
@@ -0,0 +1,201 @@
+
+
+
+html, body {
+    height: 100%;
+    min-height: 100%;
+}
+
+
+#headCityName {
+    font-family: 'Courier New', Courier, monospace;
+    font-size: 95px;
+    color:slategray;
+    opacity: .8;
+    margin-top: 0px;
+
+}
+
+
+#headCityName.sunny {
+    color: rgb(201, 232, 201);
+}
+
+#headCityName.cloudy {
+    color: #b79478;
+}
+
+#headCityName.rainy {
+    color: lightgrey; 
+}
```

I'd recommend standardizing on one color format:

```suggestion
#headCityName.sunny {
    color: #c9e8c9;
}

#headCityName.cloudy {
    color: #b79478;
}

#headCityName.rainy {
    color: #d3d3d3; 
}
```